docs-react-native/react-native/docs/ios/remote-notification-support.md

```diff
@@ -95,79 +76,141 @@ At this point everything should still be running normally. This is the final ste
 ```
 <!-- <Vimeo id="remote-notification-support-3" caption="Step 3 - Edit NotificationService.m" /> -->
 
-Before, moving to the next step, run the app and check it builds successfully – make sure you have the correct target selected. 
+## Mutate the content with Notifee:
+> 🔗 [Apple Docs: Modifying content in newly delivered notifications](https://developer.apple.com/documentation/usernotifications/modifying-content-in-newly-delivered-notifications#Implement-your-extensions-handler-methods)
 
-## Edit the payload
-Now everything is setup in your app, you can alter your notification payload in two ways:
+#### Objective C:
 
```

Comment:
Should we direct people to make sure they have the right import?

```objective-c
#import "NotifeeExtensionHelper.h"
```

Reply:
Good idea, I also add the imports in the example codes, however it's also mentioned here:
https://github.com/invertase/notifee/blob/aa5336b32f9d8fd47d3cfa46e4e1650d8ce4937a/docs-react-native/react-native/docs/ios/remote-notification-support.md?plain=1#L60-L65